metadata/transportation/road_centerlines

```diff
@@ -0,0 +1,82 @@
+# Title
+
+Utah Roads
+
+## ID
+
+a6cf0b81-a99e-4ec3-a9b7-7372ae6a6cbb
+
+## Brief Summary
+
+The latest geospatial information for all roads and highways in Utah.
+
+## Summary
+
+Utah Roads is a multi-purpose statewide roads dataset for cartography, routing, and range-based address location. This dataset is also used as the base geometry for deriving the geospatial representation of UDOT's advanced highway linear referencing system ([ALRS](https://gis.utah.gov/products/sgid/transportation/highway-routes-lrs/)) as well as [UGRC's network analysis datasets](https://gis.utah.gov/products/sgid/transportation/).
+
+## Description
+
+This dataset includes information such as the road name, type, speed limit, and infrastructure such as bike lanes. To see specific information about the data model including field names and descriptions, data types, coded value domains, and data model updates, see the [data model spreadsheet](https://docs.google.com/spreadsheets/d/1jQ_JuRIEtzxj60F0FAGmdu5JrFpfYBbSt3YzzCjxpfI/edit#gid=811360546) and the corresponding tabs for update notes. The color-coded section titled "Addressing" is primarily maintained by local stewards, whereas the section titled "DOT/FHWA" is primarily maintained by UDOT. UGRC spatially assigns fields in the section titled "Placename/Area", based on current SGID polygon boundaries.
```

Comment:
part of me thinks we remove this wording "and data model updates". I'm not convinced that info is valuable that is for our users. Yes, it's in the spreadsheet, but maybe not worth the real estate in our metadata. feel free to convince me otherwise.

Reply:
In my experience, limited as it may be, knowing the general "when it was last updated" information is usually enough. I think most people who see a spreadsheet included with a dataset are after field names and descriptions, so perhaps that is what we should focus on in the metadata. I'm also open to being convinced otherwise.

Reply:
yeah, i vote that we remove the wording, "and data model updates". but, i don't feel super strong about it, so, i'll leave it up to you. thanks!